app/config/templates/site.php

```diff
@@ -473,7 +473,7 @@ function getFramework(string $frameworkEnum, array $overrides)
         'frameworks' => [
             getFramework('FLUTTER', [
                 'providerRootDirectory' => './',
-                'buildCommand' => 'bash build.sh',
+                'buildCommand' => 'bash prepare-env.sh && flutter build web',
```

Comment:
We also need to update `providerVersion` to 0.2.*, because I released it as 0.2.0, since it has breaking changes